sr_unix/ydbenv.mpt

```diff
@@ -240,6 +239,7 @@ FileTypeEnsure(envvar,followlink)
 	. set env(envvar,"path")=$zsearch(env(envvar),$increment(zstmp))
 	. if $zlength(env(envvar,"path")) do
 	. . set env(envvar,"type")=$$FileType(env(envvar,"path"),+$get(followlink))
+	. . zsy "mkdir -p "_env(envvar,"path")_"/r"
```

Comment:
It's a hack, but we can live with it for now. As I overengineered ydbenv.mpt for what it needs to do, a solution that fits with the way it is engineered would be more complexity than it's worth. So, this is fine.

Reply:
I did think about putting a FileTypeEnsure for each directory in the path (created at line 93) and have it create it if it doesn't exist, but thought that was a level of overkill that wasn't necessary here.